packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -21,7 +21,7 @@ import {Damper, SETTLING_TIME} from './Damper.js';
 
 
 export type InteractionPolicy = 'always-allow'|'allow-when-focused';
-export type TouchMode = 'rotate'|'scroll'|'zoom';
+export type TouchMode = null|((event: TouchEvent) => void);
```

Comment:
I don't understand why, but somehow your change has made it so that iOS can no longer scroll the element. A vertical swipe with one touch doesn't move the model, so it's being filtered, but somehow not going through to the browser. Do you have an iOS device or simulator you could debug this on? If not, I can look into it.

Reply:
Uh oh... that's a bummer.
I'm afraid I do not. Please, do

Reply:
Looks like your latest changes have fixed everything! I can't  repro any problematic behavior on iOS  or  Android at this  point.